core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -117,6 +119,8 @@ class OpenMPInternal {
     return m_pool[i];
   }
 
+  bool get_level() const { return m_level; }
```

Comment:
I guess this is necessary when refactoring `execute_in_serial`?

Reply:
yes ...